plupload.module

```diff
@@ -302,7 +302,8 @@ function plupload_element_pre_render($element) {
  * Returns the path to the plupload library.
  */
 function _plupload_library_path() {
-  $path = config_get('plupload.settings','library_path');
+  // preset path to library now that it is included in module.
+  $path = 'modules/plupload/lib/plupload';
```

Comment:
You should use `backdrop_get_path` to get the module location instead of hard coding it.

Reply:
Yes, good point. I have modified the PR (I hope!).

I have not yet removed the _plupload_library_path() function because it is called four times in plupload.install and doesn't need to be fixed at the moment.

Reply:
Doesn't hurt to leave it but it should also be easy to replace. Wherever `_plupload_library_path` is called replace with `backdrop_get_path('module', 'plupload').'/lib/plupload'`

Reply:
I would prefer to define the library path in only one place so have modified the PR to introduce a new constant PLUPLOAD_LIBRARY_PATH.